lxd/main_callhook.go

```diff
@@ -66,6 +233,26 @@ func (c *cmdCallhook) Run(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("This must be run as root")
 	}
 
+	if hook == "startmountns" {
+		if len(hooksFiles) == 0 {
+			return fmt.Errorf("Missing required hooks files argument")
+		}
+
+		if c.devicesRootFolder == "" {
+			return fmt.Errorf("Missing required --devicesRootFolder <folder> flag")
```

Comment:
`<path>` or `<directory>` not `<folder>` (folder is a Windows concept).

Reply:
fixed